sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/CertificateClientTest.java

```diff
@@ -364,6 +347,15 @@ public void cancelCertificateOperation(HttpClient httpClient, CertificateService
         });
     }
 
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("getTestParameters")
+    public void cancelCertificateCreationOperation(HttpClient httpClient, CertificateServiceVersion serviceVersion) {
+        createCertificateClient(httpClient, serviceVersion);
+        cancelCertificateCreationOperationRunner((certName) -> {
+            client.cancelCertificateOperation(certName);
+        });
```

Comment:
We probably should simulate an actual scenario like firing the creation of a certificate and then cancelling the operation like the above method does. Of course, in this case we would use the `cancelCertificateOperation` method instead of cancelling through the poller just like you did here.